### DISCUSSION TITLE
Feat: Allow not sorting tags

### DISCUSSION THREAD

**Discussion by marctorsoc** [langfuse/langfuse#2848] [Ideas]
### Describe the feature or potential improvement

First, some appreciation for release this open source. This is heavily increasing my productivity 👏 

Now, my request. I see that tags are sorted automatically within each trace. In other words, if I tag a trace with ["b", "c", "a"] in the traces view, I'll see ["a", "b", "c"]. I'm not sure what is the benefit of sorting them, but I'm pretty sure most users would like to keep the order of the tags. Could we have some option to keep the order?

FWIW, I checked the code but my Typescript skills are still not good enough to find where this is done. Otherwise, I'd add a PR myself. I also checked the python code in case it's sorted there, but also could not find it.  If anyone can point me out to the source, I'll do my best to add a PR for this. But also it'd be good to have confirmation from maintainers that they're open to the change.

**Comment by marcklingen** [comment]
thanks for the feedback, let's track overall interest for this here. Generally open for any contributions to change this behavior in order to keep the order of tags!
